.github/workflows/test.yml

```diff
@@ -17,16 +17,17 @@ jobs:
     name: Build and test on Ubuntu
     runs-on: ubuntu-latest
     steps:
+      - name: Check out code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+
       - name: Set up Go 1.22
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@f111f3307d8850f501ac008e886eec1fd1932a34 # v5
         with:
-          go-version: '1.22'
+          go-version-file: 'go.mod'
+          check-latest: true
```

Comment:
gotta flip it here too

Reply:
I did, I think you're looking at an older push?